simmechanics/data/icub2_5/conf/MTB/gazebo_icub_torso_mtb.ini

```diff
@@ -0,0 +1,21 @@
+[include "../gazebo_icub_robotname.ini"]
+
+[WRAPPER]
+# name of the wrapper device to be instatiated by the factory
+device analogServer
+# rate of output streaming from ports as a ms period
+period 10
+# output port name
+name /${gazeboYarpPluginsRobotName}/torso/inertialMTB
+
+[PART_DRIVER]
+# name of the part driver device to be instatiated by the factory
+device gazebo_inertialMTB
+# part to which the sensors are attached
+# (the predefined number of sensors depends on this parameter)
+robotPart torso
+# ordered list of enabled sensors
+enabledSensors (iCub::chest::chest_mtb_acc_9b7
+				iCub::chest::chest_mtb_acc_9b8
+				iCub::chest::chest_mtb_acc_9b9
+				iCub::chest::chest_mtb_acc_9b10)
```

Comment:
Not blocking, but for the future: always avoid tabs, because you never know with which width they are rendered (for example, I guess in your editor they are visualized with four spaces, while on GitHub they are visualized as 8 spaces.